src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BedSearchService.kt

```diff
@@ -92,27 +100,58 @@ class BedSearchService(
     )
   }
 
+  @Suppress("detekt:CyclomaticComplexMethod")
   fun findTemporaryAccommodationBeds(
     user: UserEntity,
-    probationDeliveryUnit: String,
+    probationDeliveryUnit: String?,
+    probationDeliveryUnits: List<String>?,
```

Comment:
i think this should be `List<UUID>?`. Or if that's not possible can we please rename it to `probationDeliveryUnitIds` to make it's use clear?